tools/buildsys/src/cache.rs

```diff
@@ -77,8 +78,20 @@ impl LookasideCache {
     /// then verifies the contents against the SHA-512 hash provided.
     fn fetch_file<P: AsRef<Path>>(url: &str, path: P, hash: &str) -> Result<()> {
         let path = path.as_ref();
-        let mut resp =
-            reqwest::blocking::get(url).context(error::ExternalFileRequestSnafu { url })?;
+        let mut headers = HeaderMap::new();
+        headers.insert(
+            USER_AGENT,
+            HeaderValue::from_static(
+                "Bottlerocket buildsys (https://github.com/bottlerocket-os/bottlerocket)",
```

Comment:
Is there an easy way to have the version from the repo included? It's typical to have a version in the UserAgent.

Reply:
It would be a little extra work, but you could use a `std::process::Command` to call out and get `git rev-parse --short HEAD` for the repo, and/or read in the `Release.toml` file to get the version string.

Reply:
If we do want this, we could read one of the environment variables predefined by `Makefile.toml`. The most appropriate for this use to me seems `BUILDSYS_VERSION_IMAGE`, expanding to e.g. `1.13.0`.

Reply:
Good call!